pkgs/stdenv/darwin/make-bootstrap-tools.nix

```
@@ -29,7 +29,7 @@ in rec {
   cctools_ = darwin.cctools;
 
   # Avoid debugging larger changes for now.
-  bzip2_ = bzip2.override (args: { linkStatic = true; });
+  bzip2_ = bzip2.override (args: { enableStatic = true; disableShared = true; });
```

Shouldn't `disableShared` instead be `enableShared`?

---

This would preserve the existing behaviour I _think_:

```suggestion
  bzip2_ = bzip2.override (args: { enableStatic = true; enableShared = false; });
```

---

I believe this is what's causing the build failure:

> error: anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-2/pkgs/tools/compression/bzip2/default.nix:1:1 called with unexpected argument 'disableShared'

https://gist.github.com/GrahamcOfBorg/a44ee77fbe87bcf23a7a1ebdb4cddc54#file-ofborg-eval-nixpkgs-unstable-jobset-L77